app/schema.json

```diff
@@ -123,14 +133,48 @@
     "viewProperties": {
       "genomePositionSearchBox": {
         "type": "object",
-        "additionalProperties": false,
-        "required": ["chromInfoServer", "chromInfoId", "visible"],
+        "title": "The Genome Position Search Box Schema",
+        "required": [
+          "autocompleteServer",
```

Comment:
The autocomplete stuff isn't actually required. You can have a genome position search box just chromInfo.

Reply:
Must have been a merge issue. Thanks for spotting!
```suggestion
```